tests/functional/test_logs.py

```diff
@@ -35,7 +35,7 @@ def setUp(self):
         registration_info = self.balena.models.device.register(self.app["id"], self.uuid)
         self.device_api_key = registration_info["api_key"]
 
-    def __collect_logs(self, timeout: int = 5, count=None):
+    def __collect_logs(self, timeout: int = 30, count=None):
```

Comment:
Because our tests run agaisnt production I think these might have slowed a bit. I will check to see if we have enabled loki read logs which could potentially have slwoed things a bit.